pkg/service/regression/regression.go

```diff
@@ -81,11 +83,16 @@ func (r *Regression) GetApps(ctx context.Context, cid string) ([]string, error)
 	return r.tdb.GetApps(ctx, cid)
 }
 
-func (r *Regression) Get(ctx context.Context, cid, appID, id string) (models.TestCase, error) {
+func sanitiseInput(s string) string {
```

Comment:
@ditsuke Thanks for the PR. How about we add some documentation on top of this method to explain why we added this. You can also reference the issue links or the security vulnerability. It'll inform future contributors especially when refactoring the code base :)

Reply:
@slayerjain thanks, I absolutely agree! Will push a godoc to go with the function

Reply:
I've added the godoc. PTAL.